tox.ini

```diff
@@ -1,31 +0,0 @@
-# Tox (http://tox.testrun.org/) is a tool for running tests
-# in multiple virtualenvs. This configuration file will run the
-# test suite on all supported python versions. To use it, "pip install tox"
-# and then run "tox" from this directory.
-
-[tox]
-envlist = py36, py37, py38, py39, pypy, pypy3
-
-[gh-actions]
-python =
-    3.6: py36
-    3.7: py37
-    3.8: py38
-    3.9: py39
-    pypy-3.7: pypy3
-
-[testenv]
-commands = python -m unittest
-deps =
-    scipy; platform_python_implementation != "PyPy"
-    numpy; platform_python_implementation != "PyPy"
-    networkx; platform_python_implementation != "PyPy"
-    pandas; platform_python_implementation != "PyPy"
-passenv = PATH
-setenv =
-    TESTING_IN_TOX=1
-
-[flake8]
```

Comment:
Can we leave the `[flake8]` section in `tox.ini`? This does not really belong to `tox` itself; these are settings for the linter that I use (`flake8`). For historical reasons, `flake8` is configured from `tox.ini`.

Reply:
It seems that you can also put them specifically in a `.flake8` file, https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations. Perhaps that clarifies the intention of this file better?

Reply:
I'll add a `.flake8` file and then merge.